azure-iot-device/azure/iot/device/provisioning/__init__.py

```diff
@@ -4,15 +4,13 @@
 human intervention, enabling customers to provision millions of devices in a secure and scalable manner.
 
 """
-from .sk_provisioning_device_client import SymmetricKeyProvisioningDeviceClient
-from .x509_provisioning_device_client import X509ProvisioningDeviceClient
+from .provisioning_device_client import ProvisioningDeviceClient
 from .security import SymmetricKeySecurityClient
 from .security import X509SecurityClient
 from .models import RegistrationResult
 
 __all__ = [
-    "SymmetricKeyProvisioningDeviceClient",
-    "X509ProvisioningDeviceClient",
+    "ProvisioningDeviceClient",
     "SymmetricKeySecurityClient",
     "RegistrationResult",
     "X509SecurityClient",
```

Comment:
You don't need to expose SymmetricKeySercurityClient or X509SecurityClient outside this module, do you?